app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/Tooltip.tsx

```diff
@@ -254,19 +240,29 @@ const ReferencesRows: React.FC<{ title?: string; sources: Reference[] }> = props
     );
 };
 
-const InfoIcon: React.FC<{ title: string }> = props => (
-    <svg {...props}>
-        <title>{props.title}</title>
-        {props.children}
-    </svg>
-);
-
 const ButtonLink = styled.button`
     margin: 0;
     padding: 0;
     font-weight: normal;
 `;
 
+const coverageDetails = {
+    notCovered: {
+        value: i18n.t("Not in Structure Coverage"),
+        className: "error",
+        title: i18n.t(
+            "This annotation is part of the full UniProt sequence but lies outside the region captured in the 3D structure (PDB) for this specific chain. It may correspond to a flexible, disordered, or missing part of the protein that was not resolved during structure determination."
+        ),
+    },
+    partiallyCovered: {
+        value: i18n.t("Partially in Structure Coverage"),
+        className: "warning",
+        title: i18n.t(
+            "This annotation is partially covered by the 3D structure (PDB) for this specific chain. The structure may capture part of this region, but additional functional or structural details extend beyond the resolved area."
+        ),
+    },
+};
+
```

Comment:
i18n.t cannot be used at root level, as the current language may not be set at this point. The typical workaround is to wrap it in a function.